akka-bench-jmh/src/main/scala/akka/stream/NewLayoutBenchmark.scala

```diff
@@ -17,8 +19,13 @@ import org.reactivestreams.{ Publisher, Subscriber, Subscription }
 @State(Scope.Benchmark)
 @OutputTimeUnit(TimeUnit.MILLISECONDS)
 @BenchmarkMode(Array(Mode.Throughput))
+ */
+// Found names but no class, trait or object is defined in the compilation unit.
+// The incremental compiler cannot record the dependency information in such case.
+// Some errors like unused import referring to a non-existent class might not be reported.
```

Comment:
Hrm this is pretty ugly... perhaps rename it to something other than `.scala` to sidestep this issue?

Reply:
Agree, Chris asked for noting more of the issue, so that's what it is :)

Reply:
@raboof `.bench`?

Reply:
yeah or `.template` or `.txt`?